Explorer/Assets/DCL/Backpack/BackpackEquipStatusController.cs

```diff
@@ -77,6 +78,9 @@ public BackpackEquipStatusController(
             this.playerEntity = playerEntity;
             this.appArgs = appArgs;
             this.inWorldWarningNotificationView = inWorldWarningNotificationView;
+
+            publishProfileChanges = !appArgs.HasFlag(AppArgsFlags.SELF_PREVIEW_BUILDER_COLLECTIONS)
```

Comment:
Just a minor thing. I think that it is better to keep this calculation in the UpdateProfileAsync method. Basically because it ensures that is calculated when it is needed, so if we need to change the appArgs load process by any reason, we ensure that we query it in the right moment. Also i think that the performance improvement is minimal.

Reply:
Tackled at https://github.com/decentraland/unity-explorer/pull/4214/commits/4005dd26a8a874af3ff45bd8990c69887d147b0b